docs/guides/admin/docs/configuration/security.user.moodle.md

```diff
@@ -36,6 +36,11 @@ After the installation, a new user with the capabilities
 Then generate a new web service token and add that user to the "Opencast web
 service" service.
 
+### Step 0
+
+Edit `etc/org.apache.karaf.features.cfg` and make sure the `opencast-moodle` feature is listed in the `featuresBoot`
```

Comment:
We don't seem to have a etc/org.apache.karaf.features.cfg file?

Also I think this should be Step 1 and the next steps renumbered. Otherwise we'll end up with Step -1, -2, ...